scripts/unit_tests.sh

```diff
@@ -68,7 +68,7 @@ echo "begin igpu_opencl_failed_tests"
 module swap opencl opencl/intel-gpu
 sudo /opt/ocl-icd/scripts/igpu_bind &> /dev/null
 clinfo -l
-CHIP_BE=opencl ctest --timeout 180 -j 8 --output-on-failure -E "`cat ./test_lists/igpu_opencl_failed_tests.txt`" | tee igpu_opencl_make_check_result.txt
```

Comment:
Why can't we run in parallel? a lot of testing was done to get CI to run in under an hour.

Reply:
This was meant to be temporary. It is now removed.